application/application/overlays/debug/kustomization.yaml

```diff
@@ -4,3 +4,7 @@ bases:
 - ../../base
 patchesStrategicMerge:
 - stateful-set.yaml
+images:
+- name: gcr.io/$(project)/application-controller
```

Comment:
Why is this parameterized by project here? Will $(project) end up getting substituted?

Reply:
We should pass over this in a followup PR.
It's parameterized by project because that's how it is declared in the place it's used.
It probably won't get substituted.